README.md

```diff
@@ -1,83 +1,102 @@
-ca_cert
-=======
+# ca_cert puppet module
 
-[![Build Status](https://travis-ci.org/pcfens/puppet-ca_cert.png?branch=master)](https://travis-ci.org/pcfens/puppet-ca_cert)
+[![Build Status](https://github.com/voxpupuli/puppet-ca_cert/workflows/CI/badge.svg)](https://github.com/voxpupuli/puppet-ca_cert/actions?query=workflow%3ACI)
+[![Release](https://github.com/voxpupuli/puppet-ca_cert/actions/workflows/release.yml/badge.svg)](https://github.com/voxpupuli/puppet-ca_cert/actions/workflows/release.yml)
+[![Puppet Forge](https://img.shields.io/puppetforge/v/puppet/ca_cert.svg)](https://forge.puppetlabs.com/puppet/ca_cert)
+[![Puppet Forge - downloads](https://img.shields.io/puppetforge/dt/puppet/ca_cert.svg)](https://forge.puppetlabs.com/puppet/ca_cert)
+[![Puppet Forge - endorsement](https://img.shields.io/puppetforge/e/puppet/ca_cert.svg)](https://forge.puppetlabs.com/puppet/ca_cert)
+[![Puppet Forge - scores](https://img.shields.io/puppetforge/f/puppet/ca_cert.svg)](https://forge.puppetlabs.com/puppet/ca_cert)
+[![License](https://img.shields.io/github/license/voxpupuli/puppet-ca_cert.svg)](https://github.com/voxpupuli/puppet-ca_cert/blob/master/LICENSE)
```

Comment:
can you also add a badge for the migration?
```
[![Donated by Phil Fenstermacher](https://img.shields.io/badge/donated%20by-Phil%20Fenstermacher-fb7047.svg)](#transfer-notice)
```

Reply:
Done :)